test/spec/modules/openxBidAdapter_spec.js

```diff
@@ -1574,7 +1570,11 @@ describe('OpenxAdapter', function () {
         expect(bid.meta.brandId).to.equal(DEFAULT_TEST_ARJ_AD_UNIT.brand_id);
       });
 
-      it('should return a brand ID', function () {
+      it('should return an adomain', function () {
+        expect(bid.meta.advertiserDomains).to.deep.equal([]);
```

Comment:
@patmmccann will this break when that field starts getting used since it will then be equal to `adUnit.adomain` or am I looking at this wrong?

Reply:
The goal here is that the mock response in their tests will have this placeholder.  I they change their response,  they'll need to follow up here to change how it is interpreted.